sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -20,6 +22,8 @@ RUN echo "deb [arch=amd64] http://debian-archive.trafficmanager.net/debian/ bull
         echo "deb [arch=amd64] http://debian-archive.trafficmanager.net/debian bullseye-backports main" >> /etc/apt/sources.list
 
 {%- if CONFIGURED_ARCH == "armhf" %}
+# for lsb_release.py says /etc/debian_version should be numeric
+RUN echo "11.0" > /etc/debian_version
```

Comment:
This file should be coming from the `base-files` package. In your build logs, can you check to see that that package is getting installed?

Reply:
```
shil@vultr:~/sonic-buildimage$ docker run -it multiarch/debian-debootstrap:arm64-buster bash
root@4bd51dcf0847:/# cat /etc/debian_version
10.11
root@4bd51dcf0847:/# exit
exit
shil@vultr:~/sonic-buildimage$ docker run -it multiarch/debian-debootstrap:arm64-bullseye bash
root@822a90cd89ab:/# cat /etc/debian_version
bullseye/sid
root@822a90cd89ab:/#
```
/etc/debian_version is changed in debian bullseye. It makes 'lsb_release -cs' command get wrong result if add bullseye-updates debian source.
For example:
```
root@8db7064a8ae0:/# cat /etc/apt/sources.list
deb http://deb.debian.org/debian bullseye main
root@8db7064a8ae0:/# cat /etc/debian_version
bullseye/sid
root@8db7064a8ae0:/# lsb_release -cs
bullseye
root@8db7064a8ae0:/# echo "deb [arch=arm64] http://deb.debian.org/debian bullseye main contrib non-free" > /etc/apt/sources.list && \
>         echo "deb-src [arch=arm64] http://deb.debian.org/debian bullseye main contrib non-free" >> /etc/apt/sources.list && \
>         echo "deb [arch=arm64] http://deb.debian.org/debian bullseye-updates main contrib non-free" >> /etc/apt/sources.list && \
>         echo "deb-src [arch=arm64] http://deb.debian.org/debian bullseye-updates main contrib non-free" >> /etc/apt/sources.list && \
>         echo "deb [arch=arm64] http://security.debian.org bullseye-security main contrib non-free" >> /etc/apt/sources.list && \
>         echo "deb-src [arch=arm64] http://security.debian.org bullseye-security main contrib non-free" >> /etc/apt/sources.list && \
>         echo 'deb [arch=arm64] http://ftp.debian.org/debian bullseye-backports main' >> /etc/apt/sources.list
root@8db7064a8ae0:/# apt-get update
Hit:1 http://deb.debian.org/debian bullseye InRelease
Get:2 http://deb.debian.org/debian bullseye-updates InRelease [39.4 kB]
Get:3 http://security.debian.org bullseye-security InRelease [44.1 kB]
Get:4 http://ftp.debian.org/debian bullseye-backports InRelease [42.1 kB]
Get:5 http://deb.debian.org/debian bullseye/non-free Sources [96.7 kB]
Get:6 http://deb.debian.org/debian bullseye/main Sources [11.4 MB]
Get:7 http://deb.debian.org/debian bullseye/contrib Sources [51.0 kB]
Get:8 http://deb.debian.org/debian bullseye/contrib arm64 Packages [48.7 kB]
Get:9 http://deb.debian.org/debian bullseye/non-free arm64 Packages [83.4 kB]
Get:10 http://deb.debian.org/debian bullseye-updates/main Sources [1376 B]
Get:11 http://deb.debian.org/debian bullseye-updates/main arm64 Packages [2304 B]
Get:12 http://security.debian.org bullseye-security/main Sources [53.8 kB]
Get:13 http://security.debian.org bullseye-security/main arm64 Packages [88.8 kB]
Get:14 http://ftp.debian.org/debian bullseye-backports/main arm64 Packages [105 kB]
Fetched 12.1 MB in 9s (1372 kB/s)
Reading package lists... Done
root@8db7064a8ae0:/# lsb_release -cs
n/a
root@8db7064a8ae0:/# echo "11.0" > /etc/debian_version
root@8db7064a8ae0:/# lsb_release -cs
bullseye
root@8db7064a8ae0:/#
```
so, compile script like this may not work.
```
RUN add-apt-repository \
           "deb [arch={{ CONFIGURED_ARCH }}] https://download.docker.com/linux/debian \
           $(lsb_release -cs) \
           stable"
```

Reply:
I modified it at lastest commit. Thank you for your suggestions.